plugins/base/macro-action-audio.cpp

```diff
@@ -398,6 +437,9 @@ MacroActionAudioEdit::MacroActionAudioEdit(
 	_syncOffset->setMaximum(20000);
 	_syncOffset->setSuffix("ms");
 
+	_track->setMinimum(1);
+	_track->setMaximum(99);
```

Comment:
Curious, aren't there only 6 tracks? Or can you actually use more, it's just not in interface?

Reply:
To be honest I had no idea how may tracks there are and just chose a random number.
As the underlying data type indicating the mixer status is an `uint32_t` I assume the maximum is 32.

Reply:
Ah ok. So this is not the same as the recording/streaming audio channels?